include/sdf/Error.hh

```diff
@@ -247,6 +247,10 @@ namespace sdf
     /// \sa Element::SetXmlPath
     public: void SetXmlPath(const std::string &_xmlPath);
 
+    /// \brief It will print the error to sdferr or throw it using
+    /// SDF_ASSERT depending on its ErrorCode.
+    public: void throwOrPrintError();
```

Comment:
Since it's not modifying anything, it should be a `const` function

Reply:
True, done in https://github.com/gazebosim/sdformat/pull/1220/commits/6483e3ab20de09fc28f83e16ed45a67297419466.